data/json/items/tool/metalworking.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "id": "crude_bronze_nail",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "crude bronze nail" },
+    "description": "A crude bronze nail.  Probably not useful for many things due to its size and shape.",
+    "ascii_picture": "nail",
+    "weight": "16 g",
+    "volume": "750ml",
```

Comment:
This is just an absurd density?

Reply:
Thanks, I'll take a more detailed look.

Reply:
I think this is OK now, sorry about that.